streams/src/test/java/org/apache/kafka/streams/integration/HighAvailabilityTaskAssignorIntegrationTest.java

```diff
@@ -138,7 +138,7 @@ private void shouldScaleOutWithWarmupTasks(final Function<String, Materialized<O
         );
 
         final String storeName = "store" + testId;
-        final String storeChangelog = appId + "-store" + testId + "-changelog";
+        final String storeChangelog = appId + "-store-changelog";
```

Comment:
The name got too long (above the max for a topic name) with the changes in this PR. As the `appId` already contains the `testId`, it seems that we could just remove it.